.circleci/config.yml

```diff
@@ -170,12 +173,61 @@ jobs:
       - store_artifacts:
           path: dist/
 
+  build_linux_test_release:
+    parameters:
+      python_version:
+        type: string
+        default: "3.9"
+      extra_env:
+        type: string
+        default: "DEV_RELEASE=1"
+    docker:
+      - image: ubuntu:20.04
+    steps:
+      - checkout
+      - run:
+          name: Build wheel
+          command: |
+            PYTHON=python<< parameters.python_version >>
+            apt-get update
+            apt-get upgrade -y
+            DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata
+            apt-get install -y apt-utils
+            apt-get install -y software-properties-common
+            add-apt-repository -y ppa:deadsnakes/ppa
+            apt-get install -y $PYTHON $PYTHON-dev $PYTHON-full
+            apt-get install -y libblas-dev liblapack-dev liblapacke-dev
+            apt-get install -y build-essential git
```

Comment:
🙈

Reply:
Haha yeah indeed. That's about 3-4 hours of banging my head against the wall trying to make `apt-get` not ask me the timezone ...

Reply:
Another 3-4 were realizing that I have to choose the platform for `auditwheel`. I feel extremely stupid about this but everybody builds on a manylinux docker and apparently you don't specify the platform there or people just omit it from all the docs that are on the internet.